src/subscription_manager/managerlib.py

```diff
@@ -259,14 +257,31 @@ def filter_subscribed_pools(self, pools, subscribed_pool_ids,
         return filtered_pools
 
 
-def list_pools(uep, consumer_uuid, facts, list_all=False, active_on=None, filter_string=None):
+def list_pools(uep, consumer_uuid, list_all=False, active_on=None, filter_string=None):
     """
     Wrapper around the UEP call to fetch pools, which forces a facts update
     if anything has changed before making the request. This ensures the
     rule checks server side will have the most up to date info about the
     consumer possible.
     """
-    facts.update_check(uep, consumer_uuid)
+    # FIXME: for testing, replace with dbus client
```

Comment:
I don't understand this comment... Is this instructions on how to test, or is this stating we are using dbus client right now for testing?